Client/Frontend/Browser/TabManager.swift

```diff
@@ -89,7 +88,7 @@ class TabManager: NSObject {
         return configuration
     }()
 
-    fileprivate let imageStore: DiskImageStore?
+    let store: TabManagerStore!
```

Comment:
This doesnt need to be a force unwrap I think.

Reply:
Can we make `store` private? 
I see some parts of the app referencing  `isRestoring`. we could make that a computed property on tabManager for those cases.